core/jvm/src/main/scala/fs2/internal/ThreadFactories.scala

```diff
@@ -0,0 +1,32 @@
+package fs2.internal
+
+import java.lang.Thread.UncaughtExceptionHandler
+import java.util.concurrent.{ Executors, ThreadFactory }
+import java.util.concurrent.atomic.AtomicInteger
+
+import scala.concurrent.ExecutionContext
+
+private[fs2] object ThreadFactories {
+  /** A `ThreadFactory` which creates daemon threads, using the given name. */
```

Comment:
do you think it is necessarry to make this private/internal. Was usefull in libraries so I couldn't repeat same boilerplate